README.md

```diff
@@ -1,180 +1,113 @@
-where `<virtual_env_path/name>` is the name of your new Python virtual environment. You can also specify its path. On Windows, replace the activate command line above with this one: `.\<virtual_env_path/name>\Scripts\activate`.
-
-### Installation hints
-
-Here are some things to consider when setting up a development environment:
-* Remember to type the period (".") at the end of the pip install command.
-* If using a virtual environment, make sure `pip` is up to date. In initial environment testing, PySide2 is installable with only the latest version of `pip`.
-* In some setups, you might need to add the qiskit-metal folder path to your PATH variable
-* Library errors when activating conda environments, or initializing jupyter notebook/lab, might indicate a conflict between python libraries in the base and sub environments. Go ahead and manually delete the library from the base environment `site-packages` folder, shown in the error message. You might need to reinstall them in the sub environment, or create a new one.
-* If Jupyter notebook has trouble finding a dll for a package that works in the new environment outside of Jupyter, then try opening Jupyter notebook from the new environment instead of from `base`
-
-### Other Common Issues
-
-For other common installation issues, please refer to the [FAQ](https://qiskit.org/documentation/metal/faq.html)
-
-### Additional steps for developers
 
-If you are planning to develop the qiskit metal codebase, you need extra packages, which you can install by running the following command instead of (or after) the previous one:
+#### Let's create a new qubit (a transmon) by creating an object of this class.
+```python
+>>> from qiskit_metal.qlibrary.qubits.transmon_pocket import TransmonPocket
+>>> q1 = TransmonPocket(design, 'Q1', options=dict(connection_pads=dict(a=dict())))
+>>> gui.rebuild()
+>>> gui.edit_component('Q1')
+>>> gui.autoscale()
 ```
-python -m pip install -r requirements-dev.txt
+#### Change options.
+```python
+>>> q1.options.pos_x = '0.5 mm'
+>>> q1.options.pos_y = '0.25 mm'
+>>> q1.options.pad_height = '225 um'
+>>> q1.options.pad_width  = '250 um'
+>>> q1.options.pad_gap    = '50 um'
+```
+#### Update the component geoemtry, since we changed the options.
+```python
+>>> gui.rebuild()
+```
+#### Get a list of all the qcomponents in QDesign and then zoom on them.
+```python
+>>> all_component_names = design.components.keys()
+>>> gui.zoom_on_components(all_component_names)
+```
+#### Closing the Qiskit Metal GUI.
+```python
+>>> gui.main_window.close()
 ```
-You'll want to also use these instructions to [setup user environment](/docs/NEW_DEVELOPER_SETUP.md)
 
+A script is available [here](https://qiskit.org/documentation/metal/tut/overview/1.1%20High%20Level%20Demo%20of%20Qiskit%20Metal.html), where we also show the overview of Qiskit Metal.
 
+## Community and Support
 
-## Authors and Citation
+#### Watch the recorded tutorials
```

Comment:
I'm a big fan of images and icons, so I was just adding a YouTube icon or image here

Reply:
Done, I misunderstood what Zlatko was saying.